docs/content/docs/recipes/teams.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Using Fluid with Microsoft Teams
+menuPosition: 1
```

Comment:
more of a tyler question, but i'd guess the teams example should be further down the list here

Reply:
I moved it to position 4, but we can change this later if needed